en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Supported modules
+---
+
+# Commercially supported modules for Silverstripe CMS 5
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to semantic version. They are covered by:
+- our [security release process](./Managing_Security_Issues) and 
+- our [major release policy](./Major_release_policy).
+
+These modules will be supported for the lifetime of Silverstripe CMS 5 in the provided major versions.
+
+## Core Silverstripe CMS modules
+
+These modules provide the core Silverstripe CMS experience. Most Silverstripe CMS projects should install all of them.
+
+Core Module | Supported versions
+-- | --
+[silverstripe/admin](https://packagist.org/packages/silverstripe/admin) | 2
```

Comment:
There is a mix of github and packagist links here. Better to just stick to one or the other.

Reply:
Should all be using Packagist now.